packages/reporter/cypress/e2e/runnables.cy.ts

```diff
@@ -24,6 +24,7 @@ describe('runnables', () => {
       cy.visit('/').then((win) => {
         win.render(Object.assign({
           runner,
+          studioEnabled: true,
```

Comment:
We should only enable studio for a sub-set of thee tests since it's not always on

Reply:
Fixed

Also made this file more type safe and made the entire file more type safe... only to realize we don't seem to run `tsc` on this package on CI.

We should go through and properly type the entire of `reporter`.